src/plugins/diagram-viewer/diagram-content.ts

```diff
@@ -75,3 +70,21 @@ export const DiagramContentModel = ToolContentModel
   }));
 
 export interface DiagramContentModelType extends Instance<typeof DiagramContentModel> {}
+
+// The migrator sometimes modifies the diagram content model so that its create 
+// method actually goes through the migrator. When that happens if the snapshot doesn't
+// have a version the snapshot will be ignored.
+// This weird migrator behavior is documented here: src/models/mst.test.ts
```

Comment:
```suggestion
// This weird migrator behavior is demonstrated here: src/models/mst.test.ts
```
The documentation is better here.